cvat-ui/src/actions/projects-actions.ts

```diff
@@ -69,6 +69,12 @@ export function getProjectsAsync(query: Partial<ProjectsQuery>): ThunkAction {
             page: 1,
             ...query,
         };
+
+        // Check if we try to retrive single project of projects list
+        if (!Object.keys(filteredQuery).includes('id')) {
+            filteredQuery.withoutTasks = true;
+        }
```

Comment:
Maybe it is better to do it on cvat-core level?

Reply:
In this case we do not need ``withoutTasks`` option. The benefit of it is that ``withoutTasks`` is use only by internal code and it is not fully clear why we need it as a part of API.

Reply:
Made `withoutTasks` parameter computable.